docs/source/interactive.rst

```diff
@@ -5,99 +5,138 @@ Dask-jobqueue is most often used for interactive processing using tools like
-           frameborder="0" allow="autoplay; encrypted-media"
-           allowfullscreen></iframe>
+      username@hostname$ jupyter lab --no-browser --ip="*" --port 8888
 
+    You might also run ``echo $HOSTNAME`` on that machine as well to see the
+    host name.
 
-Dask Dashboard with Jupyter
----------------------------
+-   ``hpc.agency.gov`` is the address that you usually use to
+    ssh into the cluster.
 
-If you are using dask-jobqueue within Jupyter, one user friendly solution to
-see the Dashboard is to use `nbserverproxy
-<https://github.com/jupyterhub/nbserverproxy>`_. As the dashboard HTTP end
-point is launched inside the same node as Jupyter, this is a great solution for
-viewing it without having to do SSH tunneling. You just need to install
-``nbserverproxy`` in the Python environment you use for launching the notebook,
-and activate it as indicated in the docs:
+So in a real example this might look like the following:
 
 .. code-block:: bash
 
-   pip install nbserverproxy
-   jupyter serverextension enable --py nbserverproxy
+    alice@login2.summit $ jupyter lab --no-browser --ip="login2" --port 8888
+    alice@laptop        $ ssh -L 8888:login2:8888 summit.olcf.ornl.gov
 
-Then, once started, the dashboard will be accessible from your notebook URL
-by adding the path ``/proxy/8787/status``, replacing 8787 by any other
-port you use or the dashboard is bind to if needed. Sor for example:
+Additionally, if port ``8888`` is busy then you may want to choose a different
+port, like ``9999``.  Someone else may be using this port, particularly if they
+are setting up their own Jupyter server on this machine.
 
- - ``http://localhost:8888/proxy/8787/status`` with the example above
- - ``http://myjupyterhub.org/user/username/proxy/8787/status`` if using
-   JupyterHub
+You can now visit ``http://localhost:8888`` on your local browser to access the
+Jupyter server.
 
-Note that if using Jupyterhub, the service admin should deploy nbserverproxy
-on the environment used for starting singleuser notebook, but each user may
-have to activate the nbserverproxy extension.
+
+Viewing the Dask Dashboard
+--------------------------
+
+When you start a Dask Jobqueue cluster you also start a Dask dashboard.  This
+dashboard is valuable to help you understand the state of your computation and
+cluster.
+
+Typically, the dashboard is served on a separate port from Jupyter, and so can
+be used whether you choose to use Jupyter or not.  If you want to open up a
+connection to see the dashboard you can do so with SSH Tunneling as described
+above.  The dashboard's default port is at ``8787``, and is configurable with
+the ``dashboard_address=`` keyword to the Dask Jobqueue cluster objects.
+
+However, Jupyter is also able to proxy the dashboard connection through the
+Jupyter server, allowing you to access the dashboard at
+``http://localhost:8888/proxy/8787/status``.  This requires no additional SSH
```

Comment:
You don't need to install `nbserverproxy` or `jupyter-server-proxy` anymore ? It is automatically bundled with jupyter?

Reply:
It is bundled with dask-labextension